client/tests/integration/roles.rs

```diff
@@ -4,7 +4,7 @@ use std::{str::FromStr as _, time::Duration};
 
 use eyre::{eyre, Result};
 use iroha_client::client::{self, Client};
-use iroha_core::prelude::*;
+use iroha_core::{prelude::*, smartcontracts::permissions::prelude::Judge};
```

Comment:
Glob import from the prelude. I know it's pointless now, but it's more consistent.

Reply:
I think it was auto-imported. Don't know why `clippy` didn't warn about redundant import